docs/_templates/sidebar/brand.html

```diff
@@ -0,0 +1,21 @@
+{#- taken from furo==2024.5.6 -#}
+    <a class="sidebar-brand{% if logo %} centered{% endif %}" href="{{ pathto(master_doc) }}">
+      {% block brand_content %}
+      {%- if logo_url %}
+      <div class="sidebar-logo-container">
+        <img class="sidebar-logo" src="{{ logo_url }}" alt="Logo"/>
+      </div>
+      {%- endif %}
+      {%- if theme_light_logo and theme_dark_logo %}
+      <div class="sidebar-logo-container">
+        <img class="sidebar-logo only-light" src="{{ pathto('_static/' + theme_light_logo, 1) }}" alt="Light Logo"/>
+        <img class="sidebar-logo only-dark" src="{{ pathto('_static/' + theme_dark_logo, 1) }}" alt="Dark Logo"/>
+      </div>
+      {%- endif %}
+      {% if not theme_sidebar_hide_name %}
+      <!-- <span class="sidebar-brand-text">{{ docstitle if docstitle else project }}</span> -->
+      {%- endif %}
+      {% endblock brand_content %}
+    </a>
+
+    <iframe id="star" src="/furo50/github-star-btn.html?user=ABD-01&repo=furo50&type=star&count=true&size=large"  allowtransparency="true" frameborder="0" scrolling="0" width="150" height="35" title="GitHub" style="padding: var(--sidebar-item-spacing-vertical) var(--sidebar-item-spacing-horizontal);"></iframe>
```

Comment:
Why not directly use https://ghbtns.com

```html
<iframe src="https://ghbtns.com/github-btn.html?user=AB-01&repo=furo50&type=star&count=true&size=large" frameborder="0" scrolling="0" width="170" height="30" title="GitHub"></iframe>
```

Reply:
Because the body background is not getting set to transparent
![image](https://github.com/user-attachments/assets/c8543e88-714e-4541-876f-61ee006a0924)